com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/CreateAppStepTest.java

```diff
@@ -235,4 +243,111 @@ protected CreateAppStep createStep() {
         return new CreateAppStep();
     }
 
+    @RunWith(Parameterized.class)
+    public static class TestWithDocker extends SyncFlowableStepTest<CreateAppStep> {
```

Comment:
Do you really need a parameterized test with JSON files? Please try to refactor it to a regular test with no JSON file input.